src/lib/TranscriptEditor/TimedTextEditor/export-adapters/index.js

```diff
@@ -0,0 +1,20 @@
+import draftToTxt from './txt/index';
+/**
+ * Adapters for Draft.js conversion
+ * @param {json} blockData - Draft.js blocks
+ * @param {string} exportFormat - the type of file supported by the available adapters
+ */
+
+const exportAdapter = (blockData, exportFormat) => {
+  switch (exportFormat) {
+    case 'draftjs':
+      return { data: JSON.stringify(blockData), ext: 'json' };
```

Comment:
This assumes that the users is going to export the json as plain text, and therefore is needed as a string, while they might want to do something else with the json programatically, eg add it into another draftJS instance for example or other operation... in any case happy to leave it as is for now and can be revisited later if the need arises.

meanwhile I would just add the pretty print option to `JSON.stringify` so that the output once downloadable is more readable.

```suggestion
      return { data: JSON.stringify(blockData, null,2), ext: 'json' };
```